reporters/SponsorReporter.js

```diff
@@ -0,0 +1,49 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+const chalkPipe = require("chalk-pipe");
+const blue = chalkPipe("blue");
+const bold = chalkPipe("bold");
+
+class SponsorReporter {
+  constructor(expiringMultiPartyClient, walletsToMonitor, priceFeed) {
+    this.empClient = expiringMultiPartyClient;
+    this.walletsToMonitor = walletsToMonitor;
```

Comment:
What's the example JSON shape of `walletsToMonitor`? Worth documenting somewhere

Reply:
for sure. right now there is no docs or comment in this initial stub.